File: app/src/main/java/com/example/myapp/WeatherData/Sys.java
package com.example.myapp.WeatherData;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Sys {
    @SerializedName("country")
    @Expose
    public String country;
    @SerializedName("sunrise")
    @Expose
    public long sunrise;
    @SerializedName("sunset")
    @Expose
    public long sunset;
}

File: app/src/main/java/com/chplalex/shaman/mvp/model/Sys.kt
package com.chplalex.shaman.mvp.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Sys(
    @SerializedName("country") @Expose
    var country: String = "",
    @SerializedName("sunrise") @Expose
    var sunrise: Long = 0,
    @SerializedName("sunset") @Expose
    var sunset: Long = 0
)